src/components/panel/panel.js

```diff
@@ -544,4 +651,188 @@ MdPanelRef.prototype._createPanel = function() {
  */
 MdPanelRef.prototype._addStyles = function() {
   this._panelContainer.css('z-index', this._config['zIndex']);
+
+  var positionConfig = this._config['position'];
+
+  if (!positionConfig) { return; }
+
+  var isAbsolute = positionConfig.getAbsolute();
+  var panelPosition = isAbsolute ? 'fixed' : 'relative';
```

Comment:
I vaguely recall `position: fixed` for this use case having some issues on one of the mobile platforms, but I don't remember any of the details. Might be something to just watch out for.